db.go

```diff
@@ -197,6 +201,9 @@ func (x *dummyUserStore) Authenticate(identity, password string, authTypeCheck A
 	if user == nil {
 		return ErrIdentityAuthNotFound
 	}
+	if user.accountLocked == true {
```

Comment:
No need to add the ` == true`. Since Go is strongly typed, it is 100% unambiguous that an `if` statement must be written against a bool.